conf/application.conf

```diff
@@ -209,6 +209,18 @@ scheduling {
       initialDelaySecs = 31
       intervalSecs = 30
     }
+    first-phase3-reminder-expiring-test-job {
+      enabled = false
+      lockId = "first-phase3-reminder-lock-coordinator"
+      initialDelaySecs = 30
+      intervalSecs = 30
+    }
+    second-phase3-reminder-expiring-test-job {
+      enabled = false
+      lockId = "second-phase3-reminder-lock-coordinator"
+      initialDelaySecs = 30
```

Comment:
Did we ever ask @Lansalo if there was a reason the phase2 ones were staggered? Or whether it was just for load distribution?